input/ig.xml

```diff
@@ -1379,6 +1387,14 @@
 			<description value="Shows a typical related person example as the father of a patient"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-relatedperson"/>
 		</resource>
+		<resource>
+			<reference>
+				<reference value="RelatedPerson/example-sex-and-gender"/>
+			</reference>
+			<name value="RelatedPerson - as family member"/>
```

Comment:
The name does not state this is for sex and gender and it should.

Reply:
Updated name to state the example is for sex and gender